clarity_ext/domain/analyte.py

```diff
@@ -68,18 +69,17 @@ def updated_rest_resource(self, original_rest_resource, udf_map, updated_fields)
         analyte_udf_map = udf_map['Analyte']
         values_by_udf_names = {analyte_udf_map[key]: self.__dict__[key]
                                for key in analyte_udf_map if key in updated_fields}
-        assigner = AssignLogger(self)
         # Retrieve fields that are updated, only these field should be included in the rest update
         for key in values_by_udf_names:
             if _updated_rest_resource.udf.get(key, None):
                 original_type = type(_updated_rest_resource.udf[key])
                 value = get_and_apply(values_by_udf_names, key, None, original_type)
-                _updated_rest_resource.udf[key] = assigner.log_assign(key, value)
+                _updated_rest_resource.udf[key] = self.assigner.log_assign(key, value)
```

Comment:
Minor: It's a little bit confusing for me to read `log` when it's something that has to do with business-logic rather than being just a straightforward logger.